lib/kcl/kcl_manager.js

```diff
@@ -53,6 +53,7 @@ KCLManager.prototype._onAction = function(action) {
   switch (actionType) {
     case 'initialize':
     case 'processRecords':
+    case 'shutdown':
```

Comment:
This case should actually be `shutdownRequested` and ditto for below. 

The most recent release of the KCL for NodeJS migrated to mirror the KCL 2.x for Java, in which the API that the user implements changes from KCL 1.x. One such change renamed `shutdown` to `shutdownRequested` to differentiate from other types of shutdown, such as `leaseLost` or `shardEnded`. See https://docs.aws.amazon.com/streams/latest/dev/kcl2-standard-consumer-java-example.html. 

Thanks for submitting this update.
It looks like that was missed here, but is included elsewhere in the release migration. See the new sample app for example: https://github.com/awslabs/amazon-kinesis-client-nodejs/blob/master/samples/basic_sample/consumer/sample_kcl_app.js#L68